sandbox/sahus/aks/aks.tf

```
@@ -0,0 +1,67 @@
+resource "azurerm_kubernetes_cluster" "sahus_sandbox_aks" {
+  lifecycle {
+    prevent_destroy = true
+  }
+  name                = "sahus-sandbox-aks-${local.cluster_id}"
+  location            = data.azurerm_resource_group.sahus_sandbox.location
+  resource_group_name = data.azurerm_resource_group.sahus_sandbox.name
+  dns_prefix          = "${local.env}-sahus-sandbox-aks-${local.cluster_id}"
+
+  default_node_pool {
+    name                 = "sahussysnp1"
+    type                 = "VirtualMachineScaleSets"
+    orchestrator_version = "1.23.8"
+    node_count           = local.aks_system_nodes_count
+    vm_size              = local.aks_system_nodes_size
+    max_pods             = 99
+    zones                = ["a", "b", "c"]
```

Zone can be 1, 2 or 3 only
```suggestion
    zones                = ["1", "2", "3"]
```

---

updated the zones